app/models/claim.rb

```diff
@@ -39,6 +39,7 @@ class Claim < ApplicationRecord
     building_society_roll_number: true,
     payroll_run_id: false,
   }.freeze
+  DEADLINE = 6.weeks
```

Comment:
`DEADLINE` is a very vague term. Can we give it a more specific name? `CHECKING_SLA_WINDOW` or something?